time     stamp

float64  p1_cursor_y
float64  p2_cursor_y
float64  reference_y

uint32   screen_index